taskcluster/fenix_taskgraph/transforms/mark_as_shipped.py

```diff
@@ -0,0 +1,28 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Apply some defaults and minor modifications to the jobs defined in the build
+kind.
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from taskgraph.transforms.base import TransformSequence
+from taskgraph.util.schema import resolve_keyed_by
+
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def make_task_description(config, jobs):
+    for job in jobs:
+        product = "Fenix"
+        version = config.params['release_version'] or "{ver}"
+        job['worker']['release-name'] = '{product}-{version}-build{build_number}'.format(
+            product=product,
+            version=version,
+            build_number=config.params.get('build_number', 1)
```

Comment:
Good call on defaulting build_bumber while relpro lands 👍

Reply:
It was also because the full graph will still use this bit of code, even when not part of ship-it taskgraph. So defaulting ensures we get "something" here thats not busted.